dvc/output/local.py

```diff
@@ -71,17 +71,18 @@ def dumpd(self):
         return ret
 
     def verify_metric(self):
-        if not self.metric:
+        if not self.metric or self.plot:
```

Comment:
Is this correct? Or, should it be `not self.plot`?

Reply:
It is correct. Checks for plots don't differ from metric.